COPY/usr/bin/evminit.sh

```diff
@@ -10,9 +10,8 @@ VMDBDIR=/var/www/miq/vmdb
 EVMLOG=$VMDBDIR/log/evm.log
 PID_DIR=$VMDBDIR/tmp/pids
 
-
 # Log to evm.log that appliance booted
-echo `date -u` 'EVMINIT   EVM Appliance Booted' >> $EVMLOG
+echo `date -u` 'EVMINIT   EVM Appliance Booted'
```

Comment:
We should probably do that for all these log lines.  I'm trying to understand why we need any of it in the evm.log specifically.

Reply:
Yeah this was actually the only one that went to a log file, the rest of the output was already going to journald from the scraped stdout:
```
[root@localhost ~]# journalctl -u evminit.service
-- Logs begin at Fri 2022-02-11 12:01:19 EST, end at Mon 2022-02-14 13:12:58 EST. --
Feb 11 12:01:26 localhost.localdomain systemd[1]: Starting "Initializes the evm environment"...
Feb 11 12:01:26 localhost.localdomain sh[1024]: Generating a RSA private key
Feb 11 12:01:27 localhost.localdomain sh[1024]: ..............................................+++++
Feb 11 12:01:27 localhost.localdomain sh[1024]: ....................+++++
Feb 11 12:01:27 localhost.localdomain sh[1024]: writing new private key to '/var/www/miq/vmdb/certs/server.cer.key'
Feb 11 12:01:27 localhost.localdomain sh[1024]: -----
Feb 11 12:01:27 localhost.localdomain sh[1073]:   PV /dev/vda5   VG vg_data         lvm2 [<13.00 GiB / 0    free]
Feb 11 12:01:27 localhost.localdomain sh[1073]:   PV /dev/vda2   VG vg_system       lvm2 [<42.00 GiB / 0    free]
Feb 11 12:01:27 localhost.localdomain sh[1073]:   Total: 2 [54.99 GiB] / in use: 2 [54.99 GiB] / in no VG: 0 [0   ]
Feb 11 12:01:27 localhost.localdomain systemd[1]: evminit.service: Succeeded.
Feb 11 12:01:27 localhost.localdomain systemd[1]: Started "Initializes the evm environment".
```

Reply:
minor nit, but suggest renaming "Initializes the evm environment" to "Initialize the evm environment" - and wondering if we should rename evm to maybe something like appliance or application instead.

Reply:
> Yeah this was actually the only one that went to a log file, 

I meant the next line too.  Taken further can we just drop the $EVMLOG env var (and the VMDBDIR at that)

Reply:
> but suggest renaming "Initializes the evm environment" to "Initialize the evm environment"

That comes from the description of the service file, we can definitely rename that but I'll put in a different PR for that since it isn't in this file.